src/CommandingOfficers/Commander.java

```diff
@@ -135,6 +135,11 @@ public void addCOModifier(COModifier mod)
     modifiers.add(mod); // Add to the list so the modifier can be reverted next turn.
   }
 
+  public void endTurn()
+  {
+    if( aiController != null ) aiController.endTurn();
+  }
```

Comment:
This'll let me fix Cinder's price interaction with Bear&Bull

Reply:
Price interaction? Doesn't B&B only liquidate his own stuff for cost?

Reply:
B&B liquidates *anyone* on its property.
The application vs enemies is probably not going to be common, but it is a non-zero factor.
B&B is (partially) me making an Urban Blight that isn't hateful to play against.